homeassistant/components/simplefin/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""The SimpleFIN integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from simplefin4py import SimpleFin
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_ACCESS_URL
+from .coordinator import SimpleFinDataUpdateCoordinator
+
+PLATFORMS: list[str] = [Platform.SENSOR]
+
+
+type SimpleFinConfigEntry = ConfigEntry[SimpleFinData]
+
+
+@dataclass
+class SimpleFinData:
+    """Class to hold SimpleFIN data."""
+
+    sf_coordinator: SimpleFinDataUpdateCoordinator
+
```

Comment:
Might as well just remove the data and just put the coordinator directly in the config entry

Reply:
Not sure i did it the way you wanted but i did move it :)